src/megatron/bridge/recipes/__init__.py

```
@@ -24,6 +24,7 @@
 from megatron.bridge.recipes.gpt import *
 from megatron.bridge.recipes.gpt_oss import *
 from megatron.bridge.recipes.llama import *
+from megatron.bridge.recipes.ministral3 import *
```

Flake8 (7.3.0)

[error] 'megatron.bridge.recipes.ministral3.*' imported but unused

(F401)